### PR TITLE
Card modal

### PR DESCRIPTION
**Pull request by imogenem**
Modal can be opened and close, and items can be edited.
Screen Shot 2022-06-24 at 3 52 45 PM

### CODE REVIEW

**Review by nyananu — APPROVED** [review]
lgtm!

File: .idea/workspace.xml [hunk 1]
```diff
@@ -2,14 +2,12 @@
 <project version="4">
```
Comment (phreid) [inline .idea/workspace.xml:2]: .idea file made it into the commit

File: src/components/CardModal.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  { id: 1, name: 'Tools' },
+  { id: 2, name: 'Outdoors' },
+  { id: 3, name: 'DIY' }
+]
+
+const LOCATIONS = [
+  { id: 0, name: 'UBC Campus' }
+]
+
+export default function CardModal (props) {
```
Comment (phreid) [inline src/components/CardModal.jsx]: EditItemModal might be more descriptive?

File: src/redux/items/items.js [hunk 3]
```diff
@@ -8,25 +9,21 @@ export const itemSlice = createSlice({
...
-        ...state,
-        items: [
-          ...state.items.slice(0, index),
-          { ...state.items[index], status: 'Borrowed' },
-          ...state.items.slice(index + 1)
-        ]
-      }
+      state.items[index].status = 'Borrowed'
+    },
+    editItem: (state, action) => {
```
Comment (phreid) [inline src/redux/items/items.js:21]: 1) there's no 'key' attribute in the items anymore, so i'm not sure this find will work
2) you're replacing the existing item in the store right? i think the reason you're getting a console error is you're not creating a new id for the new item. see `addItem`

Comment (imogenem) [reply src/redux/items/items.js:21]: yup, fixed it! will push soon

**Pull request merged**